index.bs

```diff
@@ -2701,13 +2701,13 @@ Council Deliberations</h5>
 	in case of a tie,
 	the [=W3C Council Chair=] determines the outcome.
 	However,
-	if the decision or proposal being objected to
+	if the decision or proposal is objected to
```

Comment:
I don't understand why `is objected to` is being suggested here. `being objected to` is the correct tense.
```suggestion
	if the decision or proposal being objected to
```